cluster/gce/gci/node.yaml

```diff
@@ -83,10 +83,10 @@ write_files:
     owner: root
     content: |
       [Unit]
-      Description=Hourly kube-logrotate invocation
+      Description=kube-logrotate invocation
 
       [Timer]
-      OnCalendar=hourly
+      OnCalendar=*:0/5
```

Comment:
If we aren't going to doc every 5 minutes then the normalized form seems easier to read/understand "*-*-* *:00/5:00"

Reply:
For systemd experts it probably is more idiomatic as its shorter. However even the systemd tooling refers to the normalized form, which seems easier (to me) to read.
`$ systemd-analyze calendar *:0/5`
` Original form: *:0/5`
`Normalized form: *-*-* *:00/5:00`
`   Next elapse: Mon 2018-12-17 11:55:00 PST`
`      (in UTC): Mon 2018-12-17 19:55:00 UTC`
`      From now: 4min 55s left `

Reply:
Applied